rusoto/credential/src/object_safe.rs

```diff
@@ -1,297 +0,0 @@
-use super::{AwsCredentials, CredentialsError, ProvideAwsCredentials};
```

Comment:
Is this replaced by some code? Looks just deleted.

Reply:
Yes, this one is supposed to be removed too.